src/SME.SGP.Aplicacao/Consultas/ConsultasAula.cs

```diff
@@ -103,6 +103,8 @@ public async Task<IEnumerable<DataAulasProfessorDto>> ObterDatasDeAulasPorCalend
 
             var componentesCurriculares = await mediator.Send(new ObterComponentesCurricularesEolPorCodigoTurmaLoginEPerfilQuery(turmaCodigo, usuarioLogado.Login, usuarioLogado.PerfilAtual, true));
 
+            var disciplinaCJ = (await mediator.Send(new ObterComponentesCurricularesDoProfessorCJNaTurmaQuery(usuarioLogado.CodigoRf))).FirstOrDefault(x => x.TurmaId == turma.CodigoTurma && 
```

Comment:
aqui seria bom colocar um '?'antes do .FirstOrDefault, para se caso não vier nenhum componente (quando for professor titular), não dar erro de ref. de objeto